actions.go

```diff
@@ -415,3 +416,77 @@ func (c *Action) Getenv(key string) string {
 // GetenvFunc is an abstraction to make tests feasible for commands that
 // interact with environment variables.
 type GetenvFunc func(key string) string
+
+// Context of current workflow
+type Context struct {
+	Event      map[string]interface{}
+	EventName  string
+	SHA        string
+	Ref        string
+	Workflow   string
+	Action     string
+	Actor      string
+	Job        string
+	RunNumber  int
+	RunID      int
+	APIURL     string
+	ServerURL  string
+	GraphqlURL string
+}
+
+// Context returns the context of current action with the payload object that triggered the workflow
+func (c *Action) Context() (*Context, error) {
```

Comment:
I'm generally a fan of keeping dependencies minimal, but have you see https://github.com/sethvargo/go-envconfig? I think that could make this a lot simpler.

Reply:
If you think its ok to add this dep, yes it will make things simper and cleaner

Reply:
Yea I think it makes things a bit cleaner and more flexible.

Reply:
done